deploy/charts/rook-ceph/values.yaml

```diff
@@ -642,3 +642,8 @@ monitoring:
   # -- Enable monitoring. Requires Prometheus to be pre-installed.
   # Enabling will also create RBAC rules to allow Operator to create ServiceMonitors
   enabled: false
+
+# scc for the Rook and Ceph daemons
```

Comment:
I would suggest keeping it as a single line comment if its not a very long sentence.  Or have multiple lines if you have multiple sentences. For example, line 85-87 above.